File: samples/ruby-server/app/models/ItineraryFares.rb

class ItineraryFares

	def initialize (basePrice, taxesPrice, totalPrice)
		@basePrice = basePrice
		@taxesPrice = taxesPrice
		@totalPrice = totalPrice
	end


	# getter
	def basePrice
	  @basePrice
	end

	# setter
	def basePrice = (value)
	  @basePrice = value
	end

	def taxesPrice 
	  @taxesPrice
	end

	def taxesPrice = (value)
	  @taxesPrice = value
	end

	def totalPrice
	  @totalPrice
	end

	def totalPrice = (value)
	  @totalPrice = value
	end

end

File: samples/java-server/src/main/java/com/farandula/models/ItineraryFares.java
package com.farandula.models;

import com.nearsoft.farandula.models.Price;

/**
 * Created by enrique on 18/05/17.
 */
public class ItineraryFares {
    private Price basePrice;
    private Price taxesPrice;
    private Price totalPrice;

    public Price getBasePrice() {
        return basePrice;
    }

    public void setBasePrice(Price basePrice) {
        this.basePrice = basePrice;
    }

    public Price getTaxesPrice() {
        return taxesPrice;
    }

    public void setTaxesPrice(Price taxesPrice) {
        this.taxesPrice = taxesPrice;
    }

    public Price getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(Price totalPrice) {
        this.totalPrice = totalPrice;
    }
}
